_rules/SC3-1-1-html-xml-lang-match.md

```diff
@@ -0,0 +1,81 @@
+---
+name: SC3-1-1-html-xml-lang-match
+
+description: |
+ The rule checks that for the `html` element, there is no mismatch between the primary language in [non-empty](#non-empty) `lang` and `xml:lang` attributes, if both are used, and that it conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
```

Comment:
Remove ", and that it conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt)", since this is for another rule.